packages/cli/src/controller/init-controller.ts

```diff
@@ -218,6 +247,80 @@ export async function prepareManifest(projectPath: string, project: ProjectSpecB
   await fs.promises.writeFile(isTs ? tsPath : yamlPath, manifestData, 'utf8');
 }
 
+export function addDotEnvConfigCode(manifestData: string): string {
+  // Find the first empty line
```

Comment:
Remove unwanted comments, seems to copied code from somewhere. Please declare variable as per naming convention.